packages/yarnpkg-core/sources/Plugin.ts

```diff
@@ -9,6 +11,7 @@ import {MessageName}
 import {Project, InstallOptions}                                                              from './Project';
 import {Resolver, ResolveOptions}                                                             from './Resolver';
 import {Workspace}                                                                            from './Workspace';
+import {Body, Method}                                                                         from './httpUtils';
```

Comment:
Should I import these from `httpUtils`, or should I hoist them from `httpUtils` into types.ts?

Reply:
Still unsure of what yall would prefer here.

Reply:
They are fine in `httpUtils`; however by convention we star-import all `*Utils.ts` files:

```suggestion
import * as httpUtils                                                                         from './httpUtils';
```

Reply:
Done